src/main/java/com/codeborne/selenide/Credentials.java

```diff
@@ -0,0 +1,34 @@
+package com.codeborne.selenide;
+
+import java.util.Base64;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public final class Credentials {
+  private String login;
+  private String password;
+
+  public Credentials(final String login, final String password) {
+    this.login = login;
+    this.password = password;
+  }
+
+  /**
+   * The resulting string is base64 encoded (YWxhZGRpbjpvcGVuc2VzYW1l).
+   *
+   * @return encoded string
+   */
+  public String encode() {
+    final byte[] credentialsBytes = combine().getBytes(UTF_8);
+    return Base64.getEncoder().encodeToString(credentialsBytes);
+  }
+
+  /**
+   * Combine credentials with a colon (login:password).
+   *
+   * @return combined credentials
+   */
```

Comment:
Do we really need javadoc for private methods?

Reply:
@asolntsev 
Yes we do.
The main point thats this object is public and method like this (I mean do some combining logic with special format) is necessary to de described in any way.

Reply:
Let's make it public when it's really needed. Currently it's not used, so there is no reason to make it public.

Reply:
Agree. Just remove javadoc since it's specially private method.